frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -120,7 +113,7 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                 )
               }
               {
-                isParticipant(currentUser) && (
+                (hasRole(currentUser, "ROLE_DRIVER") || hasRole(currentUser, "ROLE_ADMIN")) && isParticipant(currentUser) && (
```

Comment:
Couldn't we just do:

```suggestion
                (hasRole(currentUser, "ROLE_DRIVER") || hasRole(currentUser, "ROLE_ADMIN")) (
```